homeassistant/components/tractive/switch.py

```diff
@@ -0,0 +1,169 @@
+"""Support for Tractive switches."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+import logging
+from typing import Any, Literal
+
+from aiotractive.exceptions import TractiveError
+
+from homeassistant.components.switch import SwitchEntity, SwitchEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import (
+    ATTR_BUZZER,
+    ATTR_LED,
+    ATTR_LIVE_TRACKING,
+    CLIENT,
+    DOMAIN,
+    SERVER_UNAVAILABLE,
+    TRACKABLES,
+    TRACKER_HARDWARE_STATUS_UPDATED,
+)
+from .entity import TractiveEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass
+class TractiveRequiredKeysMixin:
+    """Mixin for required keys."""
+
+    method: Literal["async_set_buzzer", "async_set_led", "async_set_live_tracking"]
+
+
+@dataclass
+class TractiveSwitchEntityDescription(
+    SwitchEntityDescription, TractiveRequiredKeysMixin
+):
+    """Class describing Tractive switch entities."""
+
+
+SWITCH_TYPES = (
```

Comment:
```suggestion
SWITCH_TYPES: tuple[TractiveSwitchEntityDescription, ...] = (
```
Not strictly necessary. I like to do it anyway.